docs/configuration/settings.md

```diff
@@ -142,6 +142,7 @@ You can configure the Kyuubi properties in `$KYUUBI_HOME/conf/kyuubi-defaults.co
 | kyuubi.engine.flink.memory                               | 1g                        | The heap memory for the Flink SQL engine. Only effective in yarn session mode.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | string   | 1.6.0 |
+| kyuubi.engine.flink.initialize.sql                       |                           | the initialize sql for flink engine. it fallback to  `kyuubi.engine.initialize.sql`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | seq      | 1.8.1 |
```

Comment:
```suggestion
| kyuubi.engine.flink.initialize.sql                       |                           | The initialize sql for flink engine. It fallback to  `kyuubi.engine.initialize.sql`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | seq      | 1.8.1 |
```

Reply:
The setting file is regenerated using the script, we should change the description of conf.

FYI: https://kyuubi.readthedocs.io/en/master/contributing/code/developer.html#append-descriptions-of-new-configurations-to-settings-md

Reply:
Thanks, After adopting your suggestion, it passed the related CI test, otherwise it won't be able to pass